js/src/wp-seo-term-scraper.js

```diff
@@ -1,10 +1,14 @@
-/* global YoastSEO, wp, wpseoTermScraperL10n, ajaxurl, tinyMCE, YoastReplaceVarPlugin, console */
+/* global YoastSEO, wpseoTermScraperL10n, ajaxurl, tinyMCE, YoastReplaceVarPlugin, console, require */
 (function( $ ) {
 	'use strict';
 
 	var snippetPreview;
 	var termSlugInput;
 
+	// Initialize the keyword tab module.
```

Comment:
`Initialize` is the wrong word for this, you're defining the variable, not initializing it.